docs/telemetry.md

```diff
@@ -54,20 +54,17 @@ The event ping contains a list of events ([see event format on readthedocs.io](h
 | Turbo mode switch clicked              | action   | change                | turbo_mode | on/off (the new value) | |
 | Pin site switch clicked                | action   | change                | pin_page   | on/off (the new value) | desktop_mode* |
 | Desktop mode switch clicked | action | change | desktop_mode | on/off (the new value) | |
-| Tile clicked                           | action   | click                 | home_tile  | bundled/custom | |
+| Tile clicked                           | action   | click                 | home_tile  | bundled/custom/youtube** | |
 | Tile removed                           | action   | remove                | home_tile  | bundled/custom | |
-| Unique tiles clicked per session**     | aggregate| click                 | home_tile  | `<int>` | |
+| Unique tiles clicked per session     | aggregate| click                 | home_tile  | `<int>` | |
+| Sessions that **do not** contain a click on  YouTube tile | aggregate | click | home_tile | session_without_youtube | |
 | Menu shown by user              | action   | user_show       | menu         | ||
 | Menu hidden by user              | action   | user_hide       | menu         | ||
 | No menu action taken              | aggregate   | no_action_taken       | menu         | ||
 
 (*)When the pin site switch is clicked, the state (on/off) of the desktop mode switch is also sent.
 
-(**)This is broken: see [#879](https://github.com/mozilla-mobile/firefox-tv/issues/879).** For overlay hidden, we only capture cases where the user closes the overlay themselves:
```

Comment:
I actually think this `**` is actually supposed to be on the "Menu hidden by user" telemetry probe, so don't remove it. The formatting of the `(**)` is also broken, I think.

Reply:
Issue #879 is closed, so I figured this isn't actually broken anymore- which is why I removed it.